test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -1899,20 +1899,32 @@ protected static Map<String, Object> getAlias(final String index, final String a
     }
 
     protected static Map<String, Object> getAsMap(final String endpoint) throws IOException {
-        return getAsMap(client(), endpoint);
+        return getAsMap(client(), endpoint, false);
+    }
+
+    protected static Map<String, Object> getAsMap(final String endpoint, final boolean ordered) throws IOException {
```

Comment:
Alternatively, this method could be made `private` and there could be another method `getAsOrderedMap`. This way the test code will be a bit cleaner as there won't be cryptic `true` argument.

Reply:
Yeah I like that more. Let's do that